### DISCUSSION TITLE
Add a way to edit domain in Mastercontainer GUI.

### DISCUSSION THREAD

**Discussion by PasserDreamer** [nextcloud/all-in-one#927] [Ideas] [answered]
In the README said:

How to easily log in to the AIO interface?
... **Note: You can change the domain/ip-address/port of the button by simply stopping the containers, visiting the AIO interface from the correct and desired domain/ip-address/port** ...

Expected:
A textbox that you can edit domain when all container stopped.

Result:
No way I can do on the management web page.

**Comment by szaimen** [comment]
Concerting to discussions since this is a feature request.

**Comment by szaimen** [comment]
Hi, I guess you are looking for this? https://github.com/nextcloud/all-in-one/discussions/362#discussioncomment-2624756

**Reply by PasserDreamer** [reply]
Yes, but if GUI has it would be appreciated.

**Comment by szaimen** [comment, marked as answer]
No, we will not add this for now but I will document how to do this in https://github.com/nextcloud/all-in-one/issues/933
